lib/ui/painting.dart

```diff
@@ -1568,6 +1568,15 @@ class Paint {
   }
 }
 
+/// The color space that an [Image] uses.
+enum ColorSpace {
+  /// The sRGB color gamut.
+  sRGB,
+  /// A color space that is backwards compatible with sRGB but can represent
+  /// colors outside of that gamut with values outside of [0..1];
+  extendedSRGB,
+}
+
```

Comment:
I think this enum and values probably deserve a lot more explanation. I think @Hixie would probably be interesting in taking a pass over this.

Reply:
FYI I'm finishing up a breaking change notice too that I planned on sharing with him for his review.